tests/test_extension.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+"""Test working of extension."""
+
+import os
+import pytest
+from tests import TEST_DOCS_SRC
+
+
+@pytest.mark.sphinx(srcdir=TEST_DOCS_SRC)
+def test_static_files_exists(app, status, warning):
+    """Test if the static files are present in the _build folder."""
+    app.build()
```

Comment:
Where is this defined?

Reply:
It is defined in sphinx testing fixtures -- https://github.com/sphinx-doc/sphinx/blob/b325a1de2b0ec5c897788781026096db60e640f4/sphinx/testing/fixtures.py#L110

Also,
https://github.com/rtfd/readthedocs-sphinx-search/pull/5/files#diff-484462fced51d1a06b1d93b4a44dd535R13